src/openzeppelin/account.cairo

```diff
@@ -177,63 +182,10 @@ mod Account {
     }
 
     #[internal]
-    fn _execute_single_call(mut call: Call) -> Span<felt252> {
+    fn _execute_single_call(mut call: Call) -> Array<felt252> {
         let Call{to, selector, calldata } = call;
-        starknet::call_contract_syscall(to, selector, calldata.span()).unwrap_syscall()
-    }
-}
-
-impl ArrayCallDrop of Drop::<Array<Call>>;
-
-impl CallSerde of Serde::<Call> {
-    fn serialize(ref output: Array<felt252>, input: Call) {
-        let Call{to, selector, calldata } = input;
-        Serde::serialize(ref output, to);
-        Serde::serialize(ref output, selector);
-        Serde::serialize(ref output, calldata);
-    }
-
-    fn deserialize(ref serialized: Span<felt252>) -> Option<Call> {
-        let to = Serde::<ContractAddress>::deserialize(ref serialized)?;
-        let selector = Serde::<felt252>::deserialize(ref serialized)?;
-        let calldata = Serde::<Array::<felt252>>::deserialize(ref serialized)?;
-        Option::Some(Call { to, selector, calldata })
-    }
-}
-
-impl ArrayCallSerde of Serde::<Array<Call>> {
-    fn serialize(ref output: Array<felt252>, mut input: Array<Call>) {
-        Serde::<usize>::serialize(ref output, input.len());
-        serialize_array_call_helper(ref output, input);
-    }
-
-    fn deserialize(ref serialized: Span<felt252>) -> Option<Array<Call>> {
-        let length = *serialized.pop_front()?;
-        let mut arr = ArrayTrait::new();
-        deserialize_array_call_helper(ref serialized, arr, length)
-    }
-}
 
-fn serialize_array_call_helper(ref output: Array<felt252>, mut input: Array<Call>) {
-    check_gas();
-    match input.pop_front() {
-        Option::Some(value) => {
-            Serde::<Call>::serialize(ref output, value);
-            serialize_array_call_helper(ref output, input);
-        },
-        Option::None(_) => {},
+        let res = starknet::call_contract_syscall(to, selector, calldata.span()).unwrap_syscall();
+        span_to_array(res)
```

Comment:
this is temporal no? we should add a comment about it

Reply:
There is a comment in line 71 targeting what needs to be updated, do you think adding another one here is helpful?